check-submodules.sh

```diff
@@ -45,5 +45,29 @@ ls /tmp/$SUBMODULE_NAME-new | wc -l
 # Create artifacts to be uploaded (if there are any)
 COMPARE=/tmp/$SUBMODULE_NAME-compare
 if [ "$(ls -A $COMPARE)" ]; then
-    tar -czvf /tmp/json-diffs-and-minimal-tests.tar.gz $COMPARE
+    # Maximum number of considered minimal tests for a metric
+    MT_THRESHOLD=30
+
+    # Array containing the considered metrics
+    METRICS=("cognitive" "sloc" "ploc" "lloc" "cloc" "blank" "cyclomatic" "halstead" "nom" "nexits" "nargs")
```

Comment:
This is a bit brittle. We should have a single place where we define all the metrics.
We could add a `--list-metrics` command to rca to list all the supported metrics and use it here.

Reply:
I agree, I would add a TODO for now and leave as-is

Reply:
@marco-c 

Don't you think that that type of command is useless except for this specific goal? Don't know if it's worth implementing it

Reply:
It'd be nice to have such a command anyway (one to list the metrics, one to list the metrics together with a short explanation about them).
It could be useful in other cases where you need to know all the available metrics.

Reply:
If you don't want to do it now, please file an issue and add a TODO as a comment here pointing to the issue

Reply:
#478